doc/article/en-US/templating-html-behaviors-introduction.md

```diff
@@ -297,7 +297,223 @@ The `@bindable` decorator isn't used when doing single value binding with a Cust
 
 Aurelia will call the `valueChanged` callback whenever the bound value changes. This gives the attribute a chance to change the background color of the element. In this example, we don't even need to use the `value` property that Aurelia has set for us.
 
-## [Inheritance with HTML Behaviors](aurelia-doc://section/7/version/1.0.0)
+## [Converting value with bindable properties](aurelia-doc://section/7/version/1.0.0)
+
+Aurelia binding system also provides a way to coerce value for bindable properties, to help simplify some scenarios. We can call them `typed bindable`. Consider a number input field, for example:
+
+<code-listing heading="number-field${context.language.fileExtension}">
+  <source-code lang="ES 2015">
+    export class NumberField {
+      @bindable label = 'Field Value'
+      @bindable.number value
+    }
+  </source-code>
+</code-listing>
+
+<code-listing heading="number-field.html">
+  <source-code lang="HTML">
+    <template>
+      <label>${label}
+        <input type='number' value.bind='value' />
+      </label>
+    </template>
+  </source-code>
+</code-listing>
+
+Notice the `@bindable.number`, it helps ensure whenever input element receives input, its value will be picked up and converted to number and then assigned to bound view-model. By default, there are 4 types of `typed bindable` decorators provided: `@bindable.string`, `@bindable.number`, `@bindable.date`, `@bindable.boolean`.
+
+`@bindable.number` is the simplified form of `@bindable({ coerce: 'number' })`. Which is designed to bring both simplicity and extendability to its usages. The equivalent of above usage with `coerce` is:
+
+<code-listing heading="number-field${context.language.fileExtension}">
+  <source-code lang="ES 2015">
+    export class NumberField {
+      @bindable label
+      @bindable({ coerce: 'number' }) value
+    }
+  </source-code>
+</code-listing>
+
+As you have noticed, `coerce` property is used to tell the decorator to find the right coercion function to convert incoming value. It can also be a function if you prefer:
+
+<code-listing heading="number-field${context.language.fileExtension}">
+  <source-code lang="ES 2015">
+    export class NumberField {
+      @bindable label
+      @bindable({ coerce: val => Number(val) }) value
```

Comment:
This can be `@bindable({ coerce: Number }) value` or is that confusing?

Reply:
thanks I'll fix it the next commit 👍